ClassicsKotlin/README.md

```diff
@@ -17,8 +17,15 @@ Pre-requisites
 Getting Started
 ---------------
 
-This sample uses the Gradle build system. To build this project, use the
-"gradlew build" command or use "Import Project" in Android Studio.
+```sh
+git clone https://github.com/android/tv-samples.git
+```
+
+- Open the ClassicsKotlin project in [Android Studio][studio].
+- Compile and deploy to your Android TV device (such as a Nexus Player).
```

Comment:
I don't think we should reference Nexus Player, but instead ADT-2 and ADT-3.

Reply:
Haha yeah, good point. I've updated it across the readme files.